roles/addons/sssd/templates/sssd.conf.j2

```diff
@@ -0,0 +1,30 @@
+[domain/LDAP]
+enumerate = true
+cache_credentials = true
+id_provider = ldap
+auth_provider = ldap
+#access_provider = ldap
+access_provider = permit
+chpass_provider = ldap
+ldap_uri = {{networks[j2_node_main_network]['services_ip']['authentication_servers']|join(',')}}
+ldap_search_base = {% for dc in domain_name.split('.') %}{% if not loop.first %},{% endif %}dc={{dc}}{% endfor %}
+
+ldap_tls_reqcert = never
+ldap_auth_disable_tls_never_use_in_production = true
```

Comment:
To enable tls, I think these 2 parameters skip it. Would worth investigating 😊

Reply:
Done